cmd/status.go

```diff
@@ -0,0 +1,41 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/dapr/cli/pkg/kubernetes"
+	"github.com/dapr/cli/pkg/print"
+	"github.com/dapr/cli/utils"
+	"github.com/gocarina/gocsv"
+	"github.com/spf13/cobra"
+)
+
+var StatusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Shows status and health of the Dapr control plane on Kubernetes",
+	Run: func(cmd *cobra.Command, args []string) {
+		status, err := kubernetes.Status()
```

Comment:
If we need to open this for other platforms then I think we should add a check here and for other flags(platforms) give some common response message ( status command not yet available ...? ) 
if flag=="kubernetes" { status, err := kubernetes.Status()}

Reply:
Very well, but since this is implemented in other CLI commands, this PR isn't the place to do this.